### ISSUE TITLE
save-resume option not working? with matrix factorization --rank option.

### ISSUE THREAD

**Issue by Lycanthrope92** [JohnLangford/vowpal_wabbit#699]
I am doing the matrix factorization example provided in the wiki.
I partitioned the training set into two parts: 
1. First set is 90% of the original training set.  
2. Second set is the other 10%.
I initially trained the model with this command on the first set:
vw -d input --save_resume -b 18 -q ui --rank 20 --l2 0.001 --learning_rate 0.015 --passes 20 --decay_learning_rate 0.97 --power_t 0 -f movielens.model --cache_file movielens.cache

Then I trained it on the second set:
vw -d input-2 --save_resume -c --passes 20 -i movielens.model -f movielens-v2.model

I evaluated both models on a test set:
First model (trained only on first set) resulted in average loss = 0.898649.
Second model (trained on both sets) resulted in average loss = 1.225201.

Is there an explanation as to why more training leads to a worse result?
Am I using the save_resume option in the wrong way?

Your help is appreciated.
Thanks.

**Comment by JohnLangford** [comment]
--save_resume doesn't do anything for --rank, because the base learning 
algorithm doesn't support this. You might want to experiment with --lrq 
or --mf which operate by reduction and (hence) do support --save_resume.

-John

**Comment by Lycanthrope92** [comment]
Thank you for the clarification.
